.github/workflows/test-libmaxminddb.yml

```diff
@@ -45,13 +45,35 @@ jobs:
         if: matrix.platform == 'ubuntu-latest'
 
       - name: Install libmaxminddb
-        run: brew install libmaxminddb
+        run: brew install libmaxminddb && brew ls --verbose libmaxminddb
         if: matrix.platform == 'macos-latest'
 
-      - name: Build with Werror and Wall
+      - name: Build with Werror and Wall (not macOS)
         run: python setup.py build
+        if: matrix.platform != 'macos-latest'
         env:
           CFLAGS: "-Werror -Wall -Wextra"
 
       - name: Test with tox (system libmaxminddb)
         run: tox
+        if: matrix.platform != 'macos-latest'
```

Comment:
I don't know if its necessarily a better strategy, but one pattern I've used for platform specific github action workarounds is to set environment variables for specific platforms. Something like:
```
- name: "Work around macos arm64 homebrew directory changes"
  if: runner.os == 'macos' and runner.arch = 'arm64'
  run: |
    echo "CFLAGS=-I/opt/homebrew/include" >> $env:GITHUB_ENV
    echo "LDFLAGS=-L/opt/homebrew/lib" >> $env:GITHUB_ENV

- name: Build with Werror and Wall
    run: python setup.py build
    env:
      CFLAGS: "${{ env.CFLAGS }} -Werror -Wall -Wextra"

- name: Test with tox (system libmaxminddb)
    run: tox
```

Reply:
That does seem a bit cleaner. I can try doing that.